### DISCUSSION TITLE
Is there a version compatibility problem between Sorry Cypress and Cypress v12.6

### DISCUSSION THREAD

**Discussion by vpkhachane** [sorry-cypress/sorry-cypress#752] [Q&A]
I get the following error message when running tests on sorry cypress with cypress version 12.6. it asks for the key. Previously it was not required when running the script with cypress 10.3

Same working fine on cypress 12.0.0 or 10.3

Is there a version compatibility problem between Sorry Cypress and Cypress ?

```
Your Record Key somek...mekey is not valid with this projectId: gya71u

It may have been recently revoked by you or another user.

Please log into Cypress Cloud to see the valid Record Keys.

https://on.cypress.io/dashboard/projects/gya71u
```

**Comment by DamienCassou** [comment]
You may want to use cypress-cloud instead of cy2: https://currents.dev/readme/integration-with-cypress/compatibility#sorry-cypress-users. Make sure not to switch to Cypress v13 though as it surely won't work.
